client/src/modules/templates/bhFilters.tmpl.html

```diff
@@ -7,31 +7,18 @@
     <!--
       TODO: these should be a custom CSS class.  Label is just too small!
```

Comment:
Not a big deal, but this comment really applies to the component's HTML, not the outside template.  It should probably be removed or placed there.